pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -182,7 +183,7 @@ type ReconcileHyperConverged struct {
 // Note:
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcileHyperConverged) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
```

Comment:
OK, this is new... But if we finally receiving the context from the SDK, we need to stop use ours. Please use it in `common.NewHcoRequest` (will require to change its signature, to add `ctx context.Context` as its first parameter).

Reply:
Done